sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -520,7 +520,7 @@ def construct_test_environment(
     fixture_request: Optional[pytest.FixtureRequest],
     test_suite_name: str = "integration_test",
     worker_id: str = "worker_id",
-    entity_key_serialization_version: int = 2,
+    entity_key_serialization_version: int = 3,
```

Comment:
@franciscojavierarceo @tokoko bumping the version to latest in the test construct here.